book/07-git-tools/sections/credentials.asc

```diff
@@ -19,9 +19,9 @@ Git has a few options provided in the box:
   The downside of this approach is that your passwords are stored in cleartext in a plain file in your home directory.
 * If you're using a Mac, Git comes with an "`osxkeychain`" mode, which caches credentials in the secure keychain that's attached to your system account.
   This method stores the credentials on disk, and they never expire, but they're encrypted with the same system that stores HTTPS certificates and Safari auto-fills.
-* If you're using Windows, you can install a helper called "`Git Credential Manager for Windows.`"
-  This is similar to the "`osxkeychain`" helper described above, but uses the Windows Credential Store to control sensitive information.
-  It can be found at https://github.com/Microsoft/Git-Credential-Manager-for-Windows[].
+* If you're using Windows, macOS, or Linux, you can install a helper called "`Git Credential Manager.`"
+  This uses platform-native data stores to control sensitive information.
+  It can be found at https://github.com/GitCredentialManager/git-credential-manager[].
```

Comment:
Links should have a proper name, or screenreaders will read out the link itself:

```
https colon slash slash github dot com slash gitcredentialmanager slash git hypen credential hyphen manager
```

Relevant quote from the [MDN Web Docs, link best practices](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#link_best_practices):

> Other tips:
> 
> - Don't repeat the URL as part of the link text — URLs look ugly, and sound even uglier when a screen reader reads them out letter by letter.

Reply:
Thanks! I'm not an AsciiDoc expert and didn't realize what keeping this pattern would result in. In the spirit of leaving things better than I found them, I'll reword this a bit, hopefully without violating the spirit of making a surgical change.

Some grepping the rest of the book informs me that this is far from the only instance of this pattern. You may want to do a broader sweep for each such occurrence.

Reply:
> Some grepping the rest of the book informs me that this is far from the only instance of this pattern. You may want to do a broader sweep for each such occurrence.

Thanks for checking if this is a problem elsewhere too!